tests/dummy/app/router.js

```diff
@@ -7,6 +7,13 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
+  this.route('donut');
```

Comment:
Should I remove the new handlebar files too and then add a new route and handlbar file for only the donut-chart in the new PR?